subiquity/ui/views/ubuntu_pro.py

```diff
@@ -290,7 +391,12 @@ def upgrade_yes_no_screen(self) -> Widget:
         +---------------------------------------------------------+
         """
 
-        excerpt = _("Upgrade this machine to Ubuntu Pro or skip this step.")
+        excerpt = _("Upgrade this machine to Ubuntu Pro for security updates"
+                    " on a much wider range of packages, until 2032. Assists"
```

Comment:
currentyear + 10?

Reply:
I feel like that would give a wrong information to the user, wouldn't it?

If somebody attempts an install of Jammy in say 2025, he would get "until 2035". Sadly, this is probably another instance of an information that should be dependent on the release being installed.

Reply:
/usr/share/distro-info/ubuntu.csv from package distro-info-data ?

Reply:
To not complicate this PR further, let's pull the number out of the string and deal with getting better numbers in a future PR.